src/local/index.js

```diff
@@ -323,6 +349,10 @@ class Local implements Side {
 
   // Delete a file or folder from the local filesystem
   destroy (doc, callback) {
+    if (inRemoteTrash(doc)) {
+      callback(null)
+      return
+    }
```

Comment:
Can you explain this case?

I have the impression that we should try to remove the file (and just don't throw an error if the file doesn't exist). For example, while cozy-desktop is uploading a big file, the user moves to the trash another file, and just after delete it permanently, this other file won't be deleted on the local disk after the upload finished.